src/main/java/com/project/deporturnos/controller/ReservaController.java

```diff
@@ -28,5 +29,15 @@ public ResponseEntity<?> getAllReservasAdmin() {
     }
 
 
+    @GetMapping(path = "/prueba")
+    @PreAuthorize("hasRole('ROLE_USER')")
```

Comment:
aca me equivoque y subi mal el rol, deberia ir asi: 
```suggestion
    @PreAuthorize("hasRole('ROLE_CLIENTE')")
```